catalog/transformer/catalog-transformer-mgmp/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -61,4 +62,13 @@
             <reference interface="ddf.catalog.data.AttributeRegistry"/>
         </argument>
     </bean>
+
+    <service ref="mgmpTransformer" interface="ddf.catalog.transform.MetacardTransformer" ranking="10">
```

Comment:
After looking at this in production, it seems more appropriate to remove the service ranking and rename the service id